Imagetospeech/image_to_speech.py

```diff
@@ -0,0 +1,30 @@
+import pytesseract
+from PIL import Image
+from gtts import gTTS
+import os
+
+
+pytesseract.pytesseract.tesseract_cmd = input(r'Enter the path for pytesseract: ')
+
+img = input(r"Enter image path: ")
+target = Image.open(img)
+text = pytesseract.image_to_string(target, config='')
+
+with open(f"./Imagetospeech/Text/text.txt",'w') as f:
+    f.write(text)
+
+file = open(r'./Imagetospeech/Text/text.txt')
+mytext = file.read().replace("\n"," ")
+language = 'en'
+output = gTTS(text=mytext, lang=language, slow=False)
+output.save('./Imagetospeech/sound/imagetospeech.mp3')
```

Comment:
```suggestion
output = gTTS(text=mytext, lang=language, slow=False)
output.save('./Imagetospeech/imagetospeech.mp3')
```
remove the sound folder as well

Reply:
@santushtisharma10 done